vslam.py

```diff
@@ -50,6 +45,10 @@ def main(config):
                         default=None,
                         help="Path to the configuration yaml file",
                         type=str)
-
+    parser.add_argument("-f",
```

Comment:
I think it's better to have a single configuration file. Please add feature as a component in the same config file